### DISCUSSION TITLE
How to get current route in view ?

### DISCUSSION THREAD

**Discussion by Barbapapazes** [adonisjs/core#3524] [Help] [answered]
In many front-end framework (https://router.vuejs.org/api/#linkactiveclass), it is possible to match a route to the url to personalize the style. 

How to do the same using Edge?

Thanks a lot!

**Comment by galaczi** [comment]
This is how I did it:

```
<a 
  href="{{url}}" 
  title="{{anchorText}}" 
  class="btn {{request.request.url === url ? 'btn--active' : ''}}" 
  role="{{role}}">{{anchorText}}
</a>
```

**Reply by Barbapapazes** [reply]
Thanks!

**Comment by WeyIin** [comment, marked as answer]
Alternatively you can use something like this too

```
{{ (request.matchesRoute('namedRoute')) ? 'tab-active' : 'tab-inactive' }}
```

**Reply by Barbapapazes** [reply]
Ok, thanks, I didn't know that request was accessible from template! I woks like a charm!

**Reply by Barbapapazes** [reply]
I've created a PR in the doc to add it!
